packages/react-sdk/src/useAllSpaces.ts

```diff
@@ -1,11 +1,6 @@
-'use client'
-
 import { useRiver } from './useRiver'
 
-/*
- * Returns a list with all space ids.
- */
-export const useSpaceList = () => {
+export const useAllSpaces = () => {
```

Comment:
Is this supposed to be the spaces that the user is joined? Then I suggest we pick one of the  following naming conventions: “useMySpaces”,  “useJoinedSpaces”, or “useUserSpaces” 

there are a bunch of things where this applies, we should pick something and stick to it. I personally like the “my” moniker, since it applies to lots of things like my settings, my channels, my device keys, my inbox, etc

Reply:
I like the `useMySomething`, although I think the ecosystem is pending towards the `useUserSomething` -- will do a quick research, thanks for your feedback!!